### PR TITLE
Delegate monitor update - Closes #915

### LINKED ISSUE #915: Delegates monitor displays forging delegates as non-forging
### Expected behavior
It should display delegates correctly.
### Actual behavior
It displays forging delegates as non-forging ones.

Screen Shot 2019-06-12 at 17 24 34

For example delegate `genesis_98` with pk `f62062b7590d46f382fb8c37a26ab0a1bd512951777aedcaa96822230727d3a1` is forging and his last block was forged at height `1420`. For some reason delegates monitor display last block of this delegate as height 60 and mark this non-forging.
```
11:16:01.155Z  INFO lisk-framework: Forged new block id: 1017494747384728372 height: 60 round: 1 slot: 9622896 reward: 0
11:38:40.675Z  INFO lisk-framework: Forged new block id: 9881220445803180116 height: 196 round: 2 slot: 9623032 reward: 0
11:40:40.334Z  INFO lisk-framework: Forged new block id: 6545595128597726017 height: 208 round: 3 slot: 9623044 reward: 0
12:07:31.486Z  INFO lisk-framework: Forged new block id: 3010980650107139657 height: 369 round: 4 slot: 9623205 reward: 0
12:22:01.648Z  INFO lisk-framework: Forged new block id: 4794130159607084947 height: 456 round: 5 slot: 9623292 reward: 0
12:43:30.502Z  INFO lisk-framework: Forged new block id: 97143543955552218 height: 585 round: 6 slot: 9623421 reward: 0
12:53:00.797Z  INFO lisk-framework: Forged new block id: 1021513493710531123 height: 642 round: 7 slot: 9623478 reward: 0
13:08:01.793Z  INFO lisk-framework: Forged new block id: 16017842995765863065 height: 732 round: 8 slot: 9623568 reward: 0
13:26:20.778Z  INFO lisk-framework: Forged new block id: 15094551287303683903 height: 842 round: 9 slot: 9623678 reward: 0
13:46:21.020Z  INFO lisk-framework: Forged new block id: 3110237837019984910 height: 962 round: 10 slot: 9623798 reward: 0
14:05:41.090Z  INFO lisk-framework: Forged new block id: 14302727201784906379 height: 1078 round: 11 slot: 9623914 reward: 0
14:14:20.202Z  INFO lisk-framework: Forged new block id: 14400927397487941692 height: 1130 round: 12 slot: 9623966 reward: 0
14:30:30.302Z  INFO lisk-framework: Forged new block id: 13066557217870484761 height: 1227 round: 13 slot: 9624063 reward: 0
14:59:30.264Z  INFO lisk-framework: Forged new block id: 10578738687536508986 height: 1401 round: 14 slot: 9624237 reward: 0
15:02:40.115Z  INFO lisk-framework: Forged new block id: 2104908599893244072 height: 1420 round: 15 slot: 9624256 reward: 0
```
### Steps to reproduce
Lisk Core version: `2.0.0-alpha.12`
Explorer version: `2.2.6-rc.0`

### PR DESCRIPTION
**Pull request by MichalTuleja**
### What was the problem?

Explorer was unstable when connected to Core 2.0

### How did I fix it?

- Network Monitor WebSocket implementation was updated
  - data are fetched during start and updated in regular intervals independently from a user activity
  - updates to geolocation logic -> data are loaded in parallel
- Delegate Monitor WebSocket implementation was updated
  - data are fetched during start and updated in regular intervals independently from a user activity
  - data are refreshed after a block is built or a round is changed
- Frontend bug for statistics was eliminated

### How to test it?

Check the delegate and network pages in the UI

### Review checklist

* The PR solves #915
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the
	[commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: sockets/delegateMonitor.js [hunk 1]
```diff
@@ -326,55 +312,57 @@ module.exports = function (app, connectionHandler, socket) {
...
+		log('debug', 'Client disconnected');
 	};
+
+	async.parallel([
+		// We only call getLastBlock on init, later data.lastBlock will be updated from getLastBlocks
+		getLastBlock,
+		getActive,
+		getRegistrations,
+		getVotes,
+		getNextForgers,
```
Comment (4miners) [inline sockets/delegateMonitor.js]: `getNextForgers` should be updated in an independent loop, as `blocks/change`/`rounds/change` events will not trigger if the slot is missed.

File: sockets/header.js [hunk 2]
```diff
@@ -90,35 +88,34 @@ module.exports = function (app, connectionHandler, socket) {
...
-		for (let i = 0; i < intervals.length; i++) {
-			clearInterval(intervals[i]);
-		}
-		intervals = [];
+		log('debug', 'Client disconnected');
 	};
+
+	async.parallel([
+		getBlockStatus,
+		getPriceTicker,
```
Comment (4miners) [inline sockets/header.js]: Same here - `getPriceTicker` should be independent of `blocks/change` event

File: sockets/delegateMonitor.js (hunk 1 above)
Comment (MichalTuleja) [reply sockets/delegateMonitor.js]: Done. Implemented additional interval-based updates when the block time is exceeded.

File: sockets/header.js (hunk 2 above)
Comment (MichalTuleja) [reply sockets/header.js]: Done. Implemented additional interval-based updates when the block time is exceeded which keeps price ticker up-to-date when forging is stuck.

**Pull request merged**